include/wil/resource.h

```diff
@@ -4181,7 +4183,9 @@ namespace wil
     typedef unique_any<HACCEL, decltype(&::DestroyAcceleratorTable), ::DestroyAcceleratorTable> unique_haccel;
     typedef unique_any<HCURSOR, decltype(&::DestroyCursor), ::DestroyCursor> unique_hcursor;
     typedef unique_any<HWND, decltype(&::DestroyWindow), ::DestroyWindow> unique_hwnd;
+#if !defined(NOUSER) && !defined(NOWH)

```

Comment:
This should be added to {shared,weak}_hhook

Reply:
It's probably good to have more of these, but the definition of `unique_hhook` is guarded with the same guards as `unique_hwnd`, so it should be fine, at least so far as not adding any new build breaks

Reply:
🤦🏻‍♂️I was looking at the wrong version of the file